airbyte-webapp/src/hooks/services/Analytics/useAnalyticsService.tsx

```diff
@@ -76,6 +76,17 @@ export const useAnalyticsIdentifyUser = (userId?: string, traits?: Record<string
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [userId]);
 };
+export const useAnalyticsSetAnonymousIdFromLocalStorage = (): void => {
+  const analyticsService = useAnalyticsService();
+  const anonymousId = localStorage.getItem("ajs_anonymous_id");
+
+  useEffect(() => {
+    if (anonymousId) {
+      analyticsService.setAnonymousId(JSON.parse(anonymousId));
```

Comment:
Here is the key method, provided by Segment to manually override an `anonymousId`

Reply:
https://segment.com/docs/connections/sources/catalog/libraries/website/javascript/identity/#override-the-default-anonymous-id-with-a-call